app/code/Magento/ConfigurableProduct/Block/Plugin/Product/Media/Gallery.php

```diff
@@ -59,7 +59,7 @@ public function afterGetOptionsMediaGalleryDataJson(
     private function getProductGallery($product)
```

Comment:
Please define argument & return types, like this:
```suggestion
    private function getProductGallery(Product $product): array
```

Reply:
Has been updated